handler/src/main/java/io/netty/handler/ssl/ApplicationProtocolNames.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.handler.ssl;
+
+/**
+ * Provides a set of protocol names used in ALPN and NPN.
+ *
+ * @see <a href="https://tools.ietf.org/html/rfc7540#section-11.1">RFC7540 (HTTP/2)</a>
+ * @see <a href="https://tools.ietf.org/html/rfc7301#section-6">RFC7301 (TLS ALPN Extension)</a>
+ * @see <a href="https://tools.ietf.org/html/draft-agl-tls-nextprotoneg-04#section-7">TLS NPN Extension Draft</a>
+ */
+public class ApplicationProtocolNames {
```

Comment:
@trustin I'm not sure I like keeping the available protocol names in a central place.  It seems like these should really be kept with their respective protocols. FYI, the `ApplicationProtocol` interface that I suggested (https://github.com/netty/netty/pull/3846/files#r31432476) has the protocol name in it.